skimage/filters/thresholding.py

```diff
@@ -1,10 +1,16 @@
+import math
 import numpy as np
 from scipy import ndimage as ndi
 from scipy.ndimage import filters as ndif
+from matplotlib import pyplot as plt
```

Comment:
I've just remembered that we have this discussion: https://github.com/scikit-image/scikit-image/issues/2103 :disappointed:.